src/hooks/persisted/usePlugins.ts

```diff
@@ -156,6 +160,9 @@ export default function usePlugins() {
 
   const updatePlugin = (plugin: PluginItem) => {
     return _update(plugin).then(_plg => {
+      if (plugin.version === _plg?.version) {
+        throw new Error('Try again in a moment.');
```

Comment:
Maybe we could translate this error message and stop accumulating hard coded strings.

Reply:
the configurations are too complex for quick usage :(

Reply:
I have provided the configuration in my pr.